addons/mail/static/src/models/attachment.js

```diff
@@ -174,6 +187,17 @@ registerModel({
             }
             return true;
         },
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsLinkPreview() {
+            const mimetype = [
+                'application/o-linkpreview-with-thumbnail',
+                'application/o-linkpreview',
+            ];
+            return mimetype.includes(this.mimetype);
```

Comment:
Better use Set?

Reply:
In theory yes, but I wouldn't use a set in JS in this case because the syntax is a pain (basically need to create the array to pass to the set constructor)... Only if there is a clear gain but here it's not worth it. Even the variable itself is useless, could directly call includes on the array instead.